tensorflow/core/common_runtime/copy_tensor.cc

```diff
@@ -231,6 +233,9 @@ void CopyTensor::ViaDMA(StringPiece edge_name, DeviceContext* send_dev_context,
     for (const RegistrationInfo& ri : *registry) {
       if (ri.sender_device_type == src_device_type &&
           ri.receiver_device_type == dst_device_type) {
+        if (DeviceFactory::IsPluggableDevice(src_device_type.type_string()) &&
```

Comment:
Can we refactor the `IsPluggableDevice` check out of the for loop? Also could you please add a TODO for me to revisit this after https://github.com/tensorflow/tensorflow/pull/43611 is resolved?

```c++
// TODO(penpornk): Revisit the lookup mechanism after PR #43611 (device alias) is resolved.
const bool src_device_is_pluggable = DeviceFactory::IsPluggableDevice(src_device_type.type_string());
for (const RegistrationInfo& ri : *registry) {
  ...
}
```

Reply:
done